versions/3.1.1.md

```diff
@@ -191,6 +191,10 @@ When parsing an OAD, JSON or YAML objects are parsed into specific Objects (such
 
 If the same JSON/YAML object is parsed multiple times and the respective contexts require it to be parsed as _different_ Object types, the resulting behavior is _implementation defined_, and MAY be treated as an error if detected.  An example would be referencing an empty Schema Object under `#/components/schemas` where a Path Item Object is expected, as an empty object is valid for both types.  For maximum interoperability, it is RECOMMENDED that OpenAPI Description authors avoid such scenarios.
 
+#### <a name="resolvingImplicitConnections"></a>Resolving Implicit Connections
+
+***TODO: In another PR***
+
```

Comment:
Could you please remove this?

```suggestion
```

Reply:
@ralfhandl it will get removed when I resolve conflicts with #3823, but in the meantime the PR markdown check fails if I don't have this here.

Reply:
I will convert this to a draft so it doesn't accidentally get merged with the placeholder.

Reply:
@ralfhandl I have taken the TODO out but left the header stub in.  Of the three ways I've tried to handle the PR checks failing when a link target doesn't exist across different PRs, that's the only one that has not provoked a complaint in review.